_posts/2018-12-24-ApiaryUI.md

```diff
@@ -0,0 +1,131 @@
+---
+title: ApiaryUI - Interactive Documentation v4
+excerpt: Introduction version 4 of Apiary Interface Documentation
+layout: post
+date: 2018-12-24 12:00:00 +0200
+author: char0n
+published: true
+comments: true
+---
+
+After months of hard work and burning the midnight oil I am very proud to announce
+the **ApiaryUI** - brand new, fully reviewed and refactored version of 
+[Apiary Interactive Documentation v3](https://help.apiary.io/tools/interactive-documentation/) (v3).
+
+ApiaryUI has already been deployed to [apiary.io](https://apiary.io/) on 20th of December 2018. 
+ApiaryUI is currently in what we call EAP (Early Access Program).
```

Comment:
EAP and GA totally sounds like corporate newspeak. Can't we rephrase into something akin to "We are making it available for everyone, but before it will become THE API documentation, we want to hear from you"...>?

Reply:
If EAP and GA is mandated by someone, I'd turn the explanations the other way around and put shortcuts into the parenthesis, not vice versa.